unified-runtime/source/loader/CMakeLists.txt

```diff
@@ -211,21 +211,10 @@ if(UR_ENABLE_SANITIZER)
         # In in-tree build, if LLVM is built with libc++, we also need to build
         # symbolizer.cpp with libc++ abi and link libc++ in.
         if(NOT UR_STANDALONE_BUILD AND LLVM_LIBCXX_USED)
-            execute_process(
-                COMMAND ${CMAKE_CXX_COMPILER} --print-file-name=libc++.a
-                OUTPUT_VARIABLE LIBCXX_PATH
-                OUTPUT_STRIP_TRAILING_WHITESPACE)
-            execute_process(
-                COMMAND ${CMAKE_CXX_COMPILER} --print-file-name=libc++abi.a
```

Comment:
the change here is because `--print-file-name` doesn't work as required here so the lookup always fails, so just fall back to the compiler's linking errors.